nautobot/dcim/navigation.py

```diff
@@ -298,6 +298,44 @@
                     ),
                 ),
             ),
+            NavMenuGroup(
+                name="Controllers",
+                weight=250,
```

Comment:
Should this be weight=300?

Reply:
That would require re-weighting of the other groups in this tab - probably worthwhile though.

Reply:
Re-weighted